lib/commands/connect/export.js

```diff
@@ -15,17 +15,21 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/export';
-    let response = yield api.request(context.auth.password, 'GET', url)
+    let connection, response;
```

Comment:
I don't think option 2 is much of an option unless we eliminate the cli.action stuff, because we'd have to make sure the output is *only* the exported JSON. So I'm inclined to go with option 3 in a new PR.